product_docs/docs/pgd/5/reference/catalogs-internal.mdx

```diff
@@ -56,7 +56,7 @@ An internal catalog table with per node configuration options.
 
 | Name                    | Type     | Description                              |
 | ----------------------- | -------- | ---------------------------------------- |
-| node_id                 | oid      | The node id                              |
+| node_id                 | oid      | The node ID                              |
```

Comment:
I made this change because in one table it has both id and IDs so to be consistent, I changed all to ID.